determined-ai/determined
# feat: add columns menu to Runs view

## Ticket
ET-114

## Description
Add Columns menu to Runs view. Should have same functionality as on Experiments view. Also includes header menu option to hide column.

## Test Plan
- Choose "Reset" option from menu. Note which options are checked.
- Deselect a checked column. It should be removed from the DataGrid.
- Select an unchecked column. It should be added to the DataGrid.
- Choose "Hide column" option from header menu for any visible column on the DataGrid. It should not longer be visible. Verify that the column menu also shows it unchecked.
- Click "Reset" again. Should show same options as before.
- Click "Show all". Should display all columns in the list.


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
